src/parser.ts

```diff
@@ -116,22 +116,32 @@ export function withCompilerOptions(
       const sourceFile = program.getSourceFile(filePath);
 
       const moduleSymbol = checker.getSymbolAtLocation(sourceFile);
+      if (!moduleSymbol) {
```

Comment:
I'm not sure if this is the right thing to do. But as far as I can tell without `moduleSymbol` the following code does not make sense, so we can exit here.

Reply:
you are right